packages/icons/icons.json

```
@@ -2193,406 +2193,406 @@
-        "tags": "graph, bar, histogram",
+        "tags": "graph, histogram, decreasing, descending trend, decreasing trend, levels, chart, decrease, lower",
         "group": "data",
         "codepoint": 62221
     },
     {
         "displayName": "Horizontal bar chart: descending",
         "iconName": "horizontal-bar-chart-desc",
-        "tags": "graph, bar, histogram",
+        "tags": "graph, histogram, increase, menu, descending, sort, decreasing, list, left align, left indent, left align",
         "group": "data",
         "codepoint": 61939
     },
     {
         "displayName": "Document: open",
         "iconName": "document-open",
-        "tags": "paper, access",
+        "tags": "paper, access, arrow, file, export, download",
         "group": "file",
         "codepoint": 61822
     },
     {
         "displayName": "Document: share",
         "iconName": "document-share",
-        "tags": "paper, send",
+        "tags": "paper, send, link new, open in new tab, open in new window, paper with arrow, export",
         "group": "file",
         "codepoint": 61823
     },
     {
         "displayName": "Distribution: horizontal",
         "iconName": "horizontal-distribution",
-        "tags": "alignment, layout, position",
+        "tags": "alignment, layout, position, lane, absolute, center, vertical, corridor, road, vehicle",
         "group": "editor",
         "codepoint": 61941
     },
     {
         "displayName": "Distribution: vertical",
         "iconName": "vertical-distribution",
-        "tags": "alignment, layout, position",
+        "tags": "alignment, layout, position, focus, centered, focus mode, road",
         "group": "editor",
         "codepoint": 62222
     },
     {
         "displayName": "Alignment: left",
         "iconName": "alignment-left",
-        "tags": "layout, position",
+        "tags": "layout, position, bar chart, histogram",
         "group": "editor",
         "codepoint": 61713
     },
     {
         "displayName": "Alignment: vertical center",
         "iconName": "alignment-vertical-center",
-        "tags": "layout, position",
+        "tags": "layout, position, volume, bars, biplane",
         "group": "editor",
         "codepoint": 61716
     },
     {
         "displayName": "Alignment: right",
         "iconName": "alignment-right",
-        "tags": "layout, position",
+        "tags": "layout, position, graph, chart",
         "group": "editor",
         "codepoint": 61714
     },
     {
         "displayName": "Alignment: top",
         "iconName": "alignment-top",
-        "tags": "layout, position",
+        "tags": "layout, position, graph, chart",
         "group": "editor",
         "codepoint": 61715
     },
     {
         "displayName": "Alignment: horizontal center",
         "iconName": "alignment-horizontal-center",
-        "tags": "layout, position",
+        "tags": "layout, position, drone, airplane, satellite, plug, statistics, shadow analysis, time-series",
```

```suggestion
        "tags": "layout, position, drone, airplane, satellite, statistics, shadow analysis",
```

am I missing something with "plug" and "time series"? it doesn't look like either of those to me.